Source/Core/Core/NetPlayServer.cpp

```diff
@@ -648,7 +648,7 @@ bool NetPlayServer::StartGame()
 	// no change, just update with clients
 	AdjustPadBufferSize(m_target_buffer_size);
 
-	g_netplay_initial_gctime = CEXIIPL::GetGCTime();
+	g_netplay_initial_gctime = (u32)Common::Timer::GetLocalTimeSinceJan1970();
```

Comment:
You sure? Based on the CEXIIPL::GetGCTime function it looks like the gamecube is good upto 2136 to me.